pyproject.toml

```diff
@@ -65,6 +65,8 @@ build-backend = "hatchling.build"
 plugins = ["pydantic.mypy"]
 strict = true
 allow_redefinition = true
+disable_error_code = ['valid-type' # Otherwise mypy will complain about @owned annotations
+                     ]
```

Comment:
I think we shouldn't disable this globally, only for files that actually contain Guppy code

Reply:
Removed and readded locally to the definitions in quantum.py and quantum_functional.py